### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by ben519** [eemeli/yaml#536] [General]
Hey, I use `yaml` for my platform [Scipress.io](https://www.scipress.io/). I was hoping I could sponsor this project in exchange for advertising my company on your README. I want to advertise my site, but I also want to say thanks for maintaining the package!

Let me know if this is possible and what it would entail. Thanks!

**Comment by eemeli** [comment]
Thanks to your prod, I finally got https://github.com/sponsors/eemeli set up for just this purpose.

**Reply by ben519** [reply]
Quick clarification before I become a sponsor.. For $25/mo it says

> Logo or name goes in the [yaml](https://github.com/eemeli/yaml) README

I would like to have my Logo in the README actually link to my website. Is that doable? Thanks!

**Reply by eemeli** [reply]
Absolutely! That was the intent; I've clarified the sponsor level text accordingly.

**Reply by ben519** [reply]
Okay, sweet! Just sponsored. Here's my logo, and the link should point to https://www.scipress.io/ . Keep up the great work on `yaml`!

![Color logo - no background](https://github.com/eemeli/yaml/assets/723099/f0cac84d-ff2b-48c0-8d77-dad4e00ea316)

**Reply by eemeli** [reply]
Your png was a bit big, so I recreated the source svg from your site's header, and added it to the README. Thank you!

Also pushed out a new release, so it's visible here as well: https://www.npmjs.com/package/yaml
